lib/perl/Genome/VariantReporting/Command/Wrappers/ModelReport.pm

```diff
@@ -43,11 +44,53 @@ sub execute {
             plan_file => $model_pair->plan_file($variant_type),
             translations_file => $model_pair->translations_file,
         );
-        Genome::VariantReporting::Command::CreateReport->execute(%params);
+        my $cmd = Genome::VariantReporting::Command::CreateReport->create(%params);
+
+        my $process = $cmd->execute();
+        $process->add_note(
+            header_text => 'creation metadata',
+            body_text => sprintf('%s report created by "genome variant-reporting wrappers model-report --model=%s"',
```

Comment:
If you'd prefer, `$self->command_name` will produce the string `genome variant-reporting wrappers model-report`.